compiler/AST/FnSymbol.cpp

```diff
@@ -1117,8 +1117,7 @@ const char* toString(FnSymbol* fn) {
         fn = fn->instantiatedFrom;
       }
 
-      if (fn->isMethod()) {
-        INT_ASSERT(fn->_this);
+      if (fn->isMethod() && fn->_this != NULL) {
```

Comment:
It's probably worth a comment here or in the PR message that we might want default expression functions to set fn->_this.